.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,6 @@
+If this is a general issue, you are encouraged to use the following guideline.
+
+#### Please paste your output of croutonversion from chroot below:
```

Comment:
Ok it sounds like ```sudo edit-chroot -al``` is a better option as the user may not be able to enter a chroot anymore if something has gone broken.